packages/redux/src/analytics/middlewares/__tests__/wishlist.test.ts

```diff
@@ -1,31 +1,28 @@
 import { actionTypes, getWishlistItem } from '../../../wishlists';
 import { getBrand, getCategory, getProduct } from '../../../entities';
-import {
-  initialReduxState,
-  mockBrandId,
-  mockCategoryId,
-  mockStore,
-  mockWishlistId,
-  mockWishlistItemId,
-  mockWishlistProductId,
-  mockWishlistSetId,
-} from '../../../../tests';
 import { logger } from '@farfetch/blackout-analytics/utils';
+import { mockStore, mockWishlistSetId } from '../../../../tests';
 import { wishlistMiddleware } from '../../';
+import { wishlistMockData } from 'tests/__fixtures__/analytics/wishlist';
 import Analytics, {
   eventTypes,
   fromParameterTypes,
 } from '@farfetch/blackout-analytics';
 import merge from 'lodash/merge';
+import type {
```

Comment:
All imports from `@farfetch/blackout-redux` inside this file should be relative-imported instead of importing directly from itself, as this file is inside that package.